docs/post-upgrade-req.asciidoc

```diff
@@ -0,0 +1,372 @@
+[[post-upgrade-req]]
+[role="xpack"]
+= Post upgrade requirements
+
+After upgrading to {stack} to version 7.9.x from a previous minor release
+(7.8.x, 7.7.x, and so on), you need to:
+
+* <<update-signal-index-mapping, Update `.siem-signals*` system index mappings>>.
+* <<enable-detections-ui, Enable access to the Detections page>>.
++
+Previously activated detection rules continue to run after upgrading, and this
+is only required to enable the UI.
+
+[discrete]
+[[update-signal-index-mapping]]
+== Update `.siem-signals*` index mapping
+
+After upgrading, you must update the `.siem-signals*` index mapping, which
+contains detection alerts. To do this:
+
+. Create a new {ref}/index-templates.html[index template] for storing existing
+detections alerts (see <<create-template>>).
+. Copy existing `.siem-signals*` detection alerts to a new index (see
```

Comment:
Should we put the `Copy existing...` after the `Deactivate...`? Otherwise couldn't you get new data in the original `.siem-signals*` index after we had copied everything to the new index but before we got a chance to deactivate the detection rules?

Reply:
@jonathan-buttner yep, corrected - thanks